docs/migrations/web/MIGRATION-v2.md

```diff
@@ -125,6 +125,17 @@ to the `.ModalDialog` element.
 
 - `.ModalDialog` → `.ModalDialog .ModalDialog--scrollable`
 
+### Modal: Custom Height
+
+The `--modal-preferred-height-mobile` and `--modal-preferred-height-tablet` custom properties were renamed.
+
+#### Migration Guide
+
+Update the custom properties in your project to use the new names:
+
+- `--modal-preferred-height-mobile` → `--modal-height`
```

Comment:
I removed the `mobile` suffix so the naming is aligned with `Grid` custom properties.